files/image_config/caclmgrd/caclmgrd

```diff
@@ -368,14 +376,19 @@ class ControlPlaneAclManager(object):
                     for ip_protocol in ip_protocols:
                         for dst_port in dst_ports:
                             rule_cmd = "ip6tables" if table_ip_version == 6 else "iptables"
-                            rule_cmd += " -A INPUT -p {}".format(ip_protocol)
+                            if ip_protocol != "any":
+                                rule_cmd += " -A INPUT -p {}".format(ip_protocol)
+                            else:
+                                rule_cmd += " -A INPUT "
```

Comment:
I think this can be simplified as follows:

```
rule_cmd += " -A INPUT"
if ip_protocol != "any":
    rule_cmd += " -p {}".format(ip_protocol)
```

Reply:
ok